guides/RestAPIGuide.md

```diff
@@ -1072,203 +1072,203 @@ The most common attachment type is 'file'.
   }
 }
 ```
 **Google Books Attachment**
-```
+```json
 {
   * "type" : "googlebook",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  "bookId" : <string>,
-  "viewUrl" : <string>,
-  "thumbUrl" : <string>,
-  "publishedDate" : <string>,
-  "pages" : <string>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  "bookId" : "<string>",
+  "viewUrl" : "<string>",
+  "thumbUrl" : "<string>",
+  "publishedDate" : "<string>",
+  "pages" : "<string>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **YouTube Attachment**
-```
+```json
 {
   * "type" : "youtube",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  "videoId" : <string>,
-  "uploader" : <string>,
-  "uploadedDate" : <date>,
-  "viewUrl" : <string>,
-  "thumbUrl" : <string>,
-  "tags" : <string>,
-  "duration" : <int>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  "videoId" : "<string>",
+  "uploader" : "<string>",
+  "uploadedDate" : "<date>",
+  "viewUrl" : "<string>",
+  "thumbUrl" : "<string>",
+  "tags" : "<string>",
+  "duration" : "<int>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Flickr Attachment**
-```
+```json
 {
   * "type" : "flickr",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  "author" : <string>,
-  "datePosted" : <date>,
-  "dateTaken" : <date>,
-  "licenseName" : <string>,
-  "viewUrl" : <string>,
-  "thumbUrl" : <string>,
-  "licenseCode" : <string>,
-  "licenseKey" : <string>,
-  "photoId" : <string>,
-  "mediumUrl" : <string>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  "author" : "<string>",
+  "datePosted" : "<date>",
+  "dateTaken" : "<date>",
+  "licenseName" : "<string>",
+  "viewUrl" : "<string>",
+  "thumbUrl" : "<string>",
+  "licenseCode" : "<string>",
+  "licenseKey" : "<string>",
+  "photoId" : "<string>",
+  "mediumUrl" : "<string>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **iTunesU Attachment**
-```
+```json
 {
   * "type" : "itunesu",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "playUrl" : <string>,
-  "trackName" : <string>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "playUrl" : "<string>",
+  "trackName" : "<string>",
   "links" : {
     "thumbnail" : ".../thumbs/[uuid]/[version]/[attach uuid]",
     "view" : ".../items/[uuid]/[version]/?attachment.uuid=[attach uuid]"
   }
 }
 ```
 **Package Resource Attachment**
-```
+```json
 {
   * "type" : "package-res",
-  "uuid" : <uuid>,
-  * "description" : <string>,
-  * "filename" : <string>,
-  "size" : <int>,
+  "uuid" : "<uuid>",
+  * "description" : "<string>",
+  * "filename" : "<string>",
+  "size" : "<int>",
```

Comment:
Shouldn't need to quote integers.

Reply:
https://github.com/equella/equella.github.io/pull/25#discussion_r234487858